### DISCUSSION TITLE
Start:Asking to Which Workbench Selector to Use

### DISCUSSION THREAD

**Discussion by Reqrefusion** [FreeCAD/FreeCAD#14014] [Discussions that will turn in to issues but need refining]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

The feature should be asked at the first start.

### Full version info

```shell
[code]
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.21.2.33771 (Git)
Build type: Release
Branch: releases/FreeCAD-0-21
Hash: b9bfa5c5507506e4515816414cd27f4851d00489
Python 3.8.10, Qt 5.15.2, Coin 4.0.1, Vtk 8.2.0, OCC 7.6.3
Locale: Turkish/Turkey (tr_TR)
Installed mods: 
  * IconThemes
  * OpenDark 2024.3.13
[/code]
```

### Subproject(s) affected?

Other (specify in description)

### Anything else?

We discussed it #13771 for a bit, but unfortunately there is no definitive conclusion at the moment, but as long as there are two workbench selectors, they should be asked at the first startup, and maybe which one is recommended for whom, etc. can be written.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by obelisk79** [comment]
I think this would be too granular, rather than such specificity there should be two layouts. Classic and Modern. Classic being the current 0.21 style default panel layout and workbench selector. And modern, overlay, split panel, TabBar. This exposes quick settings which show two opposing UI arrangements users could quickly switch between and decide which is closest to their preference. Additionally, it will act as a brief introduction into a broad array of customization options which will be at their disposal later on once they get that far in their user journey. 

Also, you aren't stating a problem as much as you're phrasing the desired solution indirectly as the problem. This isn't conducive to a construction dialog about what the best way to determine what should be presented to new users in the start dialog.

**Comment by Reqrefusion** [comment]
> I think this would be too granular, rather than such specificity there should be two layouts. Classic and Modern. Classic being the current 0.21 style default panel layout and workbench selector. And modern, overlay, split panel, TabBar. This exposes quick settings which show two opposing UI arrangements users could quickly switch between and decide which is closest to their preference. Additionally, it will act as a brief introduction into a broad array of customization options which will be at their disposal later on once they get that far in their user journey.
> 
> Also, you aren't stating a problem as much as you're phrasing the desired solution indirectly as the problem. This isn't conducive to a construction dialog about what the best way to determine what should be presented to new users in the start dialog.

What you are saying is a more comprehensive version of the solution I suggested. Rather than making such a suggestion, I suggested a simpler incremental solution. You said something about what the problem is here. Whatever problem TabBar was the solution to, that's the problem here. It feels shabby.

**Comment by furgo16** [comment]
> ## Problem description
> The feature should be asked at the first start.

That does not describe a problem. Could you please elaborate on the motivation for this proposal? That is, what the actual issue is and what the proposal that tries to address it is.

Also, would the [Discussions](https://github.com/FreeCAD/FreeCAD/discussions) section not be a better place for this conversation?

**Reply by Reqrefusion** [reply]
There is one modern feature and one old one. Currently the old feature is the default and an extra effort is required to turn on the modern one. A first-time user should be able to easily unlock this modern feature. Otherwise he won't even be aware of its existence. What was wanted was clearly there because there was a choice between the things you wanted in the discussion. I have explained the reasons I mentioned here at length in the discussion, and I find it wrong to bring them back to the discussion without looking at them.
